project-base/app/tests/FrontendApiBundle/Functional/Login/LoginTest.php

```diff
@@ -16,12 +16,60 @@ class LoginTest extends GraphQlTestCase
      */
     protected TokenFacade $tokenFacade;
 
+    public function testAccessTokenIsNotValidAfterPasswordChange(): void
+    {
+        $graphQlType = 'Login';
+
+        $response = $this->getResponseContentForGql(
+            __DIR__ . '/graphql/LoginMutation.graphql',
+            $this->getDefaultCredentials(),
+        );
+        $responseData = $this->getResponseDataForGraphQlType($response, $graphQlType);
+
+        $this->assertArrayHasKey('tokens', $responseData);
+        $this->assertArrayHasKey('accessToken', $responseData['tokens']);
+        $accessToken = $responseData['tokens']['accessToken'];
+
+        $clientOptions = ['HTTP_X-Auth-Token' => sprintf('Bearer %s', $accessToken)];
+        $this->configureCurrentClient(null, null, $clientOptions);
+
+        $response = $this->getResponseContentForGql(
+            __DIR__ . '/../_graphql/query/CurrentCustomerUserQuery.graphql',
+        );
+        $responseData = $this->getResponseDataForGraphQlType($response, 'currentCustomerUser');
+
+        $this->assertArrayHasKey('email', $responseData);
+        $this->assertSame($this->getDefaultCredentials()['email'], $responseData['email']);
+
+        $response = $this->getResponseContentForGql(
+            __DIR__ . '/graphql/ChangePasswordMutation.graphql',
+            [
+                'email' => $this->getDefaultCredentials()['email'],
+                'oldPassword' => $this->getDefaultCredentials()['password'],
+                'newPassword' => 'user124',
+            ],
+        );
+        $responseData = $this->getResponseDataForGraphQlType($response, 'ChangePassword');
```

Comment:
I would like to discuss the behavior from a UX point of view - is it ok to logout the current customer when he changes his own password? This might be out of the scope of this issue, however, I would expect the password change would logout the customer from all the devices except the current one :thinking: Let's talk about it, please

Reply:
This is a current behavior. 
But prior to this change, it behaves a little more weirdly, as the user is logged out after the access token expires.
We may discuss the possible improvements, but I would like to separate it to the new PR